app-frontend/src/app/components/tokenItem/tokenItem.html

```diff
@@ -0,0 +1,44 @@
+<div class="token-item-container">
+  <div class="list-group-right">
+    <div class="form-group all-in-one" ng-if="$ctrl.token.name">
+      <input type="text" class="form-control"
+             ng-model="$ctrl.newName"
+             ng-disabled="!$ctrl.editing">
+      <input type="text"
+             ng-if="!$ctrl.editing"
+             class="form-control"
+             style="cursor: text"
+             ng-value="$ctrl.token.id" disabled>
+      <button class="btn btn-link"
+              ng-if="!$ctrl.editing"
+              disabled>
```

Comment:
It doesn't work. We need to bring in a library for it. There are a couple other areas of the application with the same button / problem. I'll make an issue for it